tests/specs/session-replay/rrweb-configuration.e2e.js

```diff
@@ -58,7 +58,7 @@ describe.withBrowsersMatching(notIE)('RRWeb Configuration', () => {
         .then(() => browser.waitForAgentLoad())
 
       const [{ request: { body } }] = await Promise.all([
-        browser.testHandle.expectBlob(),
+        browser.testHandle.expectBlob(10000),
```

Comment:
Just curious, why are we adding a timeout on these expects when we are not expecting them to fail? The timeout was original added so we could expect a timeout and not have to wait 65 seconds. I guess this is just so we don't have to wait 65 seconds to find out the harvest didn't happen?

Reply:
yep exactly, i had some failures earlier when i was building this and this was just added to fail quicker